homeassistant/components/nam/sensor.py

```diff
@@ -2,31 +2,63 @@
 from __future__ import annotations
 
 from datetime import timedelta
-from typing import Any
-
-from homeassistant.components.sensor import ATTR_STATE_CLASS, SensorEntity
+import logging
+from typing import cast
+
+from homeassistant.components.sensor import (
+    ATTR_STATE_CLASS,
+    DOMAIN as PLATFORM,
+    SensorEntity,
+)
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import ATTR_DEVICE_CLASS, ATTR_ICON
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 from homeassistant.util.dt import utcnow
 
 from . import NAMDataUpdateCoordinator
-from .const import ATTR_ENABLED, ATTR_LABEL, ATTR_UNIT, ATTR_UPTIME, DOMAIN, SENSORS
+from .const import (
+    ATTR_ENABLED,
+    ATTR_LABEL,
+    ATTR_UNIT,
+    ATTR_UPTIME,
+    DOMAIN,
+    MIGRATION_SENSORS,
+    SENSORS,
+)
 
 PARALLEL_UPDATES = 1
 
+_LOGGER = logging.getLogger(__name__)
+
 
 async def async_setup_entry(
     hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Add a Nettigo Air Monitor entities from a config_entry."""
     coordinator: NAMDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
 
+    # Due to the change of the attribute name of two sensora, it is necessary to migrate
```

Comment:
the attribute name of two sensors

Reply:
https://github.com/home-assistant/core/pull/52182